backend/curation/api/curation-api.yml

```diff
@@ -578,6 +707,28 @@ components:
         type: string
       nullable: true
       example: ["patient", "seqBatch"]
+    census_info_dataset:
```

Comment:
"Cell Census" references must not appear in the swagger. 

Cell Census is only one example of a bulk downloader. The solution needs to be general purpose.